assets/js/ViewerApp.tsx

```diff
@@ -79,23 +95,28 @@ class ViewerApp extends React.Component<
       }));
     });
 
-    channel.on('new_sign_configs_state', (state) => {
+    signsChannel.on('new_sign_configs_state', (state) => {
       this.setState({ signConfigs: state });
     });
 
-    channel.on('new_configured_headways_state', (state) => {
+    headwaysChannel.on('new_configured_headways_state', (state) => {
       this.setState({ configuredHeadways: state });
     });
 
-    channel.on('new_alert_state', (alertState) => {
+    signsChannel.on('new_alert_state', (alertState) => {
```

Comment:
Ah, sorry! I don't think this existed when I wrote up the ticket, but I think alert state should have its own channel as well. Associating an alert with an ID is part of signs, so that can stay, but the `new_alert_state` is independently coming from the Alert.State GenServer.